qiskit/dagcircuit/dagcircuit.py

```diff
@@ -1102,6 +1118,7 @@ def edge_weight_map(wire):
             new_node._node_id = new_node_index
             new_node.op.condition = condition
             self._multi_graph[new_node_index] = new_node
+            self._increment_op(old_node.op)
```

Comment:
`new_node.op`? (I can see `old_node.op` is not wrong, though)

Reply:
Yeah, I could have gone either way `old_node.op` == `new_node.op` here so it didn't matter. I can change it if you think it's more clear to use `new_node.op`